storage/innobase/row/row0sel.cc

```diff
@@ -2886,7 +2886,7 @@ void row_sel_field_store_in_mysql_format_func(
 
     row_sel_field_store_in_mysql_format(mysql_rec + templ->mysql_col_offset,
                                         templ, rec_index, field_no, data, len,
-                                        &prebuilt->compress_heap, sec_field_no);
+                                        &prebuilt->blob_heap, sec_field_no);
```

Comment:
this change of cmpress_heap to blob_heap is not relevant in 5.7? :thinking:

Reply:
Yes, it is. In 5.7 we pass `prebuilt` as the parameter, then `blob_heap` is allocated conditionally in `row_decompress_column`.
Here we pass heap pointer, which, again, is conditionally allocated in `row_decompress_column`

Reply:
:+1: